chainer/optimizers/adam.py

```diff
@@ -128,7 +137,7 @@ class AdamRule(optimizer.UpdateRule):
     def __init__(self, parent_hyperparam=None,
                  alpha=None, beta1=None, beta2=None, eps=None,
                  eta=None, weight_decay_rate=None, amsgrad=None,
-                 adabound=None, final_lr=None, gamma=None):
+                 adabound=None, radam=None, final_lr=None, gamma=None):
```

Comment:
Let's make this argument keyword-only.

Reply:
with keyword only you mean to place `radam=None` after `, *, radam=None`? as specfified in [PEP3102](https://www.python.org/dev/peps/pep-3102/).
Doing it this way will break Jenkins since python2 support is not disabled yet.

Reply:
We can either wait for Jenkins fix, or use `chainer.utils.argument.parse_kwargs`.

Reply:
Maybe we should wait for #7650 to unify the three selections with the same style?